drivers/tee/optee/core.c

```diff
@@ -357,6 +357,26 @@ static bool optee_msg_api_uid_is_optee_api(optee_invoke_fn *invoke_fn)
 	return false;
 }
 
+static void optee_msg_get_os_revision(optee_invoke_fn *invoke_fn)
+{
+	union {
+		struct arm_smccc_res smccc;
+		struct optee_smc_calls_revision_result result;
+	} res = {
+		.result = {
+			.reserved0 = 0
```

Comment:
Since we're now actually using this member it should have a proper name.

Reply:
I was worried about breaking backwards compatibility, but since (1) it is in-kernel code and (2) no one is supposed to use a reserved field, it sounds reasonable indeed.